chartjs/colors.py

```diff
@@ -34,8 +34,8 @@ def next_color(color_list=COLORS):
     >>> next(g)
     [84, 83, 79]
     """
-    step = 1
+    step = 0
     while True:
         for color in color_list:
-            yield map(lambda base: (base + step) % 256, color)
+            yield list(map(lambda base: (base + step) % 256, color))
```

Comment:
Why would you expand the generator here?

Reply:
I just saw your comments on this. From the docstring, it looked like you wanted to return a list rather than a generator here. Since `map` used to return a list, I thought this was the intent.